### DISCUSSION TITLE
CI failures

### DISCUSSION THREAD

**Discussion by pjc0247** [DefinitelyTyped/DefinitelyTyped#58948] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58525

I made a simple change but there're several test failures even not related with my changes.

Can someone help this out?

Thanks.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
@pjc0247 Please could you force push empty commit?

**Reply by pjc0247** [reply]
thanks! it works now
